src/index.js

```diff
@@ -128,13 +129,22 @@ class Service {
 
 function raw (service, method, params) {
   // handle client methods like indices.create
-  const [meth, ext] = method.split('.');
-
-  if (typeof ext !== 'undefined') {
-    return service.Model[meth][ext](params);
+  const [primaryMethod, secondaryMethod] = method.split('.');
+
+  if (typeof service.Model[primaryMethod] === 'undefined') {
+    return Promise
+      .reject(new Error(`There is no query method ${primaryMethod}.`));
+  } else if (
+    secondaryMethod &&
+    typeof service.Model[primaryMethod][secondaryMethod] === 'undefined'
+  ) {
```

Comment:
Add logic control to throw warning when the primary method is ok but secondary method screws up.